### PR TITLE
Initial nft pallet

### LINKED ISSUE #6: Initial NFT pallet integration
We need to support NFT's. To do so we would integrate ORML NFT pallet - https://github.com/open-web3-stack/open-runtime-module-library/tree/master/nft

### PR DESCRIPTION
**Pull request by korzewski**
Closes #6

### CODE REVIEW

File: pallets/nft/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[package]
+authors = ['Anmol']
+description = 'FRAME pallet template for defining custom runtime logic.'
+edition = '2018'
+license = 'Unlicense'
+name = 'pallet-nft'
+readme = 'README.md'
+repository = 'https://github.com/substrate-developer-hub/substrate-node-template/'
+version = '0.1.0'
```
Comment (aelesbao) [inline pallets/nft/Cargo.toml:9]: Let's keep the header consistent with the other `Cargo.toml` files (like the [kiities-pallet](https://github.com/anmolnetwork/anmol-node/blob/master/pallets/kitties/Cargo.toml#L2-L10))

**Review by nblogist — CHANGES_REQUESTED** [review]
@korzewski check fails can you please check the issue.

**Comment by korzewski** [comment]
All request changes seems to be done, please review again

**Review by nblogist — APPROVED** [review]
Looks good! @aelesbao please add your review.

**Review by aelesbao — APPROVED** [review]
lgtm!

**Pull request merged**